terraform/gcp/modules/oslogin/general.auto.tfvars

```diff
@@ -0,0 +1,17 @@
+project_id = "projectsigstore-staging"
```

Comment:
Do we need this file?

Reply:
Oops, no that was a test file. Thanks for catching. Removed.